GFDL-CM4-dtr.yaml computed DTR fails validation with invalid DTR range

The cleaned DTR output computed for GFDL-CM4 fails to validate. The error given is that the DTR ranges are invalid. The data run were historical, ssp245, ssp585 — all of which failed.

Workflow run: https://argo.cildc6.org/archived-workflows/default/59cb861b-431e-4832-8751-cc13bdf1ef3d

Sample of logged error from the historical run failure:
```
Validating gs://clean-b1dbca25/cmip6/CMIP/NOAA-GFDL/GFDL-CM4/historical/r1i1p1f1/day/dtr/gr1/v20180701.
zarr
Traceback (most recent call last):
  File "/argo/staging/script", line 44, in <module>
    tasks = dask.compute(*tasks)
  File "/opt/conda/lib/python3.9/site-packages/dask/base.py", line 570, in compute
    results = schedule(dsk, keys, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/dask/threaded.py", line 79, in get
    results = get_async(
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 507, in get_async
    raise_exception(exc, tb)
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 315, in reraise
    raise exc
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 220, in execute_task
    result = _execute_task(task, data)
  File "/opt/conda/lib/python3.9/site-packages/dask/core.py", line 119, in _execute_task
    return func(*(_execute_task(a, cache) for a in args))
  File "/argo/staging/script", line 31, in clear_memory_intensive_tests
    _test_dtr_range(d, v)
  File "/opt/dodola/dodola/core.py", line 719, in _test_dtr_range
    assert (ds[var].min() > 0) and (
AssertionError: diurnal temperature range values are invalid
```

Blocking progress on #401

--- a/workflows/templates/qualitycontrol-check-cmip6.yaml
+++ b/workflows/templates/qualitycontrol-check-cmip6.yaml
@@ -40,7 +40,7 @@ spec:
           - name: data
           - name: time
       script:
-        image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:dev
+        image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:0.13.0
         command: [python]
         source: |
           import dask
@@ -73,7 +73,7 @@ spec:
               if v == "tasmin" or v == "tasmax":
                   _test_temp_range(d, v)
               if v == "dtr":
-                  _test_dtr_range(d, v)
+                  _test_dtr_range(d, v, data_type)
               if v == "dtr" or v == "pr":
                  _test_negative_values(d, v)
               if v == "pr":

--- a/workflows/templates/qualitycontrol-check-cmip6.yaml
+++ b/workflows/templates/qualitycontrol-check-cmip6.yaml
@@ -43,6 +43,9 @@ spec:
         image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:0.13.0
         command: [python]
         source: |
+          # Running this as a script rather than a dodola command as workaround to 
+          # https://github.com/ClimateImpactLab/dodola/issues/126
+          
           import dask
           import numpy as np
           from dodola.core import (_test_for_nans, _test_variable_names, _test_timesteps, _test_temp_range, _test_dtr_range, _test_negative_values, _test_maximum_precip)

